operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -115,25 +115,14 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=5
 	Persistence *Persistence `json:"persistence,omitempty"`
 
+	// Settings for Central DB, which is responsible for data persistence.
```

Comment:
Is this ready for user consumption at this point? If not, perhaps it deserves a mention here?

Reply:
added tech preview note

Reply:
Ooo magic, thanks. I had no idea what htose did